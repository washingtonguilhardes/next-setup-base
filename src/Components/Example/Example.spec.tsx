import React from 'react';

import { render, cleanup } from '@testing-library/react';

import Example from './Example';

afterEach(cleanup);

describe('Example', () => {
  it('render', () => {
    const { container } = render(<Example />);
    expect(container.getElementsByTagName('ul')).toHaveLength(1);
  });
  it('snapshot', () => {
    const { container } = render(
      <Example>
        <span>ok</span>
      </Example>
    );
    expect(container).toMatchSnapshot();
  });
});
