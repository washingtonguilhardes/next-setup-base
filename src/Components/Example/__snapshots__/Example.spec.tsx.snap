// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example snapshot 1`] = `
<div>
  <div>
    <ul>
      <li>
        <span>
          ok
        </span>
      </li>
    </ul>
  </div>
</div>
`;
